src/sql/parts/query/editor/charting/chartView.ts

```diff
@@ -209,8 +209,13 @@ export class ChartView extends Disposable implements IPanelView {
 	}
 
 	private buildOptions() {
-		dispose(this.optionDisposables);
-		this.optionDisposables = [];
+		// The first element in the disposables list is for the chart type: the master dropdown that controls other option controls.
+		// whiling rebuilding the options we should not dispose it, otherwise it would react to the theme change event
+		if (this.optionDisposables.length > 1) {
```

Comment:
Might make more sense to just make disposable a option when creating the controls and set to false for this one

Reply:
the chart type select box needs to be disposed when the view is getting disposed, we need to keep it tracked.